### DISCUSSION TITLE
How to delete records after disconnecting in many-to-one relation?

### DISCUSSION THREAD

**Discussion by ngoc199** [prisma/prisma#10794] [Q&A]
My models look like this

```prisma
model Article {
  id          String  @id
  title        String
  isPublished Boolean @default(false)

  comments Comment[]
}

model Comment {
  id    String @id
  content  String
  articleId String

  article Article @relation(fields: [articleId], references: [id])
}
```

I am using `upsert` to update the article, but it only has `connectOrCreate` to update the comments, which does not delete the removed comments at all.

How could I `disconnect` and `delete` the removed comment?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
